packages/story-editor/src/components/canvas/pageSideMenu.js

```diff
@@ -71,6 +74,12 @@ function PageSideMenu() {
   const quickActions = useQuickActions();
 
   const isZoomed = zoomSetting !== ZOOM_SETTING.FIT;
+  const insertElement = useInsertElement();
+  const handleAddText = (evt) => {
+    evt.stopPropagation();
+    insertElement('text', DEFAULT_PRESET);
+    trackEvent('library_text_quick_action');
```

Comment:
This should follow the naming convention for other Quick Actions tracking events.

e.g.

```js
trackEvent('quick_action', {
  name: ACTIONS.INSERT_TEXT.trackingEventName,
}
```

Reply:
Done